### DISCUSSION TITLE
SSL Handshake Issue when Upgrading from 1.10.x to 1.11.x with Self-Signed Certificates

### DISCUSSION THREAD

**Discussion by Bornegrim** [vercel/turborepo#7317] [Help]
### Issue:
Encountering a "Sending fatal alert BadCertificate" error when upgrading from version 1.10.x to 1.11.x.

### Context:
We have a custom cache (ducktors/turborepo-remote-cache) hosted on an AKS cluster. The cache servers utilize self-signed certificates within a chain. In version 1.10.x, everything worked seamlessly. However, upon upgrading to 1.11.x, the SSL handshake fails with the mentioned "Sending fatal alert BadCertificate" error.

### Temporary Solution:
Using the `--go-fallback` flag resolves the issue, but this flag is no longer available in version 1.12.x.

### Configuration:
Custom cache hosted on AKS cluster
Certificates in the chain are self-signed
Cache server's ingress uses its own certificate plus an intermediate certificate
Root certificate is installed on the local machine
### Expected Behavior:
Successful SSL handshake without the need for the `--go-fallback` flag, consistent with the behavior observed in version 1.10.x.
### Question:
Has anyone else encountered a similar SSL handshake issue when upgrading from version 1.10.x to 1.11.x in the presence of self-signed certificates? Additionally, are there any suggestions or insights on resolving this problem without the use of the `--go-fallback` flag in version 1.12.x? Your input would be valuable in troubleshooting and addressing this issue.

**Comment by mattwood-nutrien** [comment]
I'm also seeing this on 1.13.2 .  It's only a warning for me, but it's persistent.

**Reply by mattwood-nutrien** [reply]
Just discovered: ```turbo telemetry disable``` which remove the warning I've been getting.
